templates/rails_helper.rb

```diff
@@ -1,19 +1,15 @@
-ENV["RAILS_ENV"] = "test"
+ENV['RAILS_ENV'] = 'test'
 
-require File.expand_path("../../config/environment", __FILE__)
+require 'spec_helper'
 
-require "rspec/rails"
-require "shoulda/matchers"
+require File.expand_path('../../config/environment', __FILE__)
 
-Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |file| require file }
+require 'rspec/rails'
+require 'shoulda/matchers'
 
-module Features
-  # Extend this module in spec/support/features/*.rb
-  include Formulaic::Dsl
-end
+Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |file| require file }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.